src/test/java/org/folio/rest/impl/GOBIIntegrationServiceResourceImplTest.java

```diff
@@ -92,13 +117,14 @@ public final void testGetGobiValidate(TestContext context) {
 
     RestAssured
       .given()
-        .header(tenantHeader)
-        .header(contentTypeHeaderJSON)
+      .header(tenantHeader)
+      .header(urlHeader)
+      .header(contentTypeHeaderJSON)
       .when()
-        .get(validatePath)
+      .get(validatePath)
       .then()
-        .statusCode(204)
-        .content(Matchers.equalTo(""));
+      .statusCode(204)
+      .content(Matchers.equalTo(""));
```

Comment:
Not a big deal, but it would be nice to have the indentation levels kept here. It is a little easier to grok when it is broken into given/when/then blocks. Sucks for auto formatting, which is probably what happened here. Eclipse and other IDEs have the ability to skip auto formatting code between `// @formatter: off` and `// @formatter: on` comments. It has to be turned on in Eclipse, so not ideal. It seems like this is a common complaint for RestAssured tests with little for solutions, but again, no big deal... it just looks like a wall of method calls. :)